usaspending_api/awards/models.py

```diff
@@ -291,6 +305,7 @@ class FinancialAssistanceAward(AwardAction):
     reporting_period_start = models.DateField(blank=True, null=True)
     reporting_period_end = models.DateField(blank=True, null=True)
     last_modified_date = models.DateField(blank=True, null=True)
+    submitted_type = models.CharField(max_length=1, blank=True, null=True, verbose_name="Submitted Type")
```

Comment:
@alexjajabah thanks for the context--will follow up to make sure we don't need a future story for this